### PR TITLE
Make the slot use all the horizontal space when not positioned at right in desktop and create description prop in header

### PR DESCRIPTION
**Pull request by wyunreal**
Pending:
- Deprecate preamount, amount, button and subtitle props from Header.

### CODE REVIEW

File: src/__stories__/header-story.tsx [hunk 1]
```diff
@@ -72,18 +74,15 @@ export const Default: StoryComponent = () => {
                             isErrorAmount={isErrorAmount}
                         />
                     }
-                    extra={
-                        withExtraContent ? (
-                            <Text3 regular>some account chart here, for example</Text3>
-                        ) : undefined
-                    }
+                    extra={withExtraContent ? <Placeholder /> : undefined}
```
Comment (atabel) [inline src/__stories__/header-story.tsx:77]: yes, better

File: src/header.tsx [hunk 2]
```diff
@@ -26,25 +26,45 @@ type RichText = string | ({text: string} & OverridableTextProps);
 type HeaderProps = {
     pretitle?: RichText;
     title?: string;
+    description?: string;
+    dataAttributes?: DataAttributes;
+    /**
+     * @deprecated This field is deprecated, please use the extra slot in the HeaderLayout component instead.
```
Comment (atabel) [inline src/header.tsx:32]: I'd remove deprecated props from the story too

Comment (yceballost) [reply src/header.tsx:32]: also from playroom snippets pls!

**Review by Winde — APPROVED** [review]

**Review by sergivillar — APPROVED** [review]

**Review by yceballost — APPROVED** [review]
nice, thanks W!

**Review by atabel — APPROVED** [review]

**Pull request merged**